ocaml/networkd/bin_db/networkd_db.ml

```diff
@@ -31,22 +31,59 @@ let _ =
     (Printf.sprintf "Usage: %s [-bridge <bridge> | -iface <interface>]" name) ;
   try
     let config = Network_config.read_config () in
-    if !bridge <> "" then
-      if List.mem_assoc !bridge config.bridge_config then (
-        let bridge_config = List.assoc !bridge config.bridge_config in
-        let ifaces =
-          List.concat_map (fun (_, port) -> port.interfaces) bridge_config.ports
-        in
-        Printf.printf "interfaces=%s\n" (String.concat "," ifaces) ;
-        match bridge_config.vlan with
+    ( if !bridge <> "" then
```

Comment:
Could the case of `!bridge = ""` not be handled as part of `List.assoc_opt` returning `Null`? I would try to reduce the deep nesting of conditions.

Reply:
This is to distinguish the cases that no `-bridge` option and  `-bridge <br> where <br> is an invalid one`.